src/include/duckdb.h

```diff
@@ -1626,6 +1628,16 @@ Creates a list value from a type and an array of values of length `value_count`
 */
 DUCKDB_API duckdb_value duckdb_create_list_value(duckdb_logical_type type, duckdb_value *values, idx_t value_count);
 
+/*!
+Creates a array value from a type and an array of values of length `value_count`
+
+* type: The type of the array
+* values: The values for the array
+* value_count: The number of values in the array
+* returns: The value. This must be destroyed with `duckdb_destroy_value`.
+*/
+DUCKDB_API duckdb_value duckdb_create_array_value(duckdb_logical_type type, duckdb_value *values, idx_t value_count);
```

Comment:
Curious why you're passing the value count here, when it's also part of the array type itself?

Reply:
No wait, the type we pass in is the child type, not the type of the array itself!